src/modules/mentors/mentors.controller.ts

```diff
@@ -46,7 +46,7 @@ export class MentorsController {
   @Get()
   @UsePipes(new PaginationPipe())
   async index(@Req() request: Request, @Query() filters: MentorFiltersDto) {
-    const userId: string = request.user.auth0Id;
+    const userId: string = request.user?.auth0Id;
```

Comment:
`userId` is marked as a string but it would be `undefined` if `request.user` is `undefined`. Can we make sure it'll always be a string?

```js
const userId: string = request.user?.auth0Id || '';
```

Reply:
@arku and I discussed this concept a bit further on Slack and concluded that manually enforcing a level of type safety  the compiler isn't enforcing may not be the best choice here.   There are at least two alternatives we should consider instead; I will make an issue and back-reference it here so we can have that conversation separately.

Reply:
The issue was added.  See below 👇  or just click into #185.